cmd/terramate/e2etests/cloud/run_cloud_deployment_test.go

```diff
@@ -151,8 +151,9 @@ func TestCLIRunWithCloudSyncDeployment(t *testing.T) {
 			cmd:      []string{HelperPathAsHCL, "echo", "${terramate.stack.path.absolute}"},
 			want: want{
 				run: RunExpected{
-					Status: 0,
-					Stdout: "/stack\n",
+					IgnoreStderr: true,
```

Comment:
Please, add the `--quiet` so the non-important stderr messages are ignored. We must keep the WARNs not affected by `--quiet` though then we can catch issues.